tests/python/sparsetir/sparse_tir_lowered_iter_scripts.py

```diff
@@ -31,7 +31,14 @@ def csrmm(
     nnz: T.int32,
 ) -> None:
     # function attr dict
-    T.func_attr({"global_symbol": "main", "tir.noalias": True, "sparse_tir_level": 1})
+    T.func_attr(
+        {
+            "global_symbol": "main",
+            "tir.noalias": True,
+            "sparse_tir_level": 1,
+            "check_invalid_binary_search": False,
```

Comment:
I suppose this annotation would not be emitted because we will only emit `"check_invalid_binary_search": True` when invalid binary search check is enabled.

Reply:
actually,`check_invalid_binary_search  ` is a sign to tell the `lower_sparse_buffer` pass whether we should consider invalid index created by the lower_iter_pass.This is to reduce the overhead.Otherwise, a sign should add to the  `lower_sparse_buffer` pass in other ways.

Reply:
I mean we can choose not to print this annotation when `check_invalid_binary_search` is not enabled.